ksql-functional-tests/src/test/resources/query-validation-tests/aslist.json

```
@@ -0,0 +1,20 @@
+{
+  "comments": [
+    "Tests covering AS_LIST functionality with generics"
+  ],
+  "tests": [
+    {
+      "name": "construct a list from two elements",
+      "statements": [
+        "CREATE STREAM TEST (a INT, b INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT as_list(a, b, 3) as l FROM TEST;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 1, "value": {"a":  1, "b":  2}, "timestamp": 0}
```

```suggestion
        {"topic": "test_topic", "value": {"a":  1, "b":  2}}
```

The `key` and the `timestamp` are just noise here - remove them.

---

Same for other QTT tests.